CONTRIBUTING.MD

```diff
@@ -0,0 +1,170 @@
+---
+title: "Getting Started"
+weight: 1
+aliases: [ "/contributor-guide" ]
+description:
+  A list of things that you should be familiar with before you
+  get started with contributing to the Camunda Cloud Documentation. 
+  This includes such things as signing the contributor License Agreement, 
+  familiarizing yourself with our Code of Conduct,
+  and more.
+---
```

Comment:
Since this is a plain markdown page I don't think we need this section anymore, that said, you may want to add a title to this page like `# Contributing Guide` and adjust the Getting Started header to an H2.

Reply:
Done in latest commit! :) Thank you for the review!